src/networks.json

```diff
@@ -532,11 +532,11 @@
     "name": "Thundercore Mainnet",
     "chainId": 108,
     "network": "mainnet",
-    "multicall": "",
+    "multicall": "0x3017086deef56679e267f67f66c4415109b7a97f",
     "rpc": [
       "https://mainnet-rpc.thundercore.com"
```

Comment:
Is there any full archive node for this network?? current one is not a full archive

Reply:
Please use the following RPC
"https://mainnet-rpc.thundercore.com/archived/SNAPSHOTEuR82a75fLYA"
"https://mainnet-rpc.thundercore.com"
"https://mainnet-rpc.thundertoken.net"
"https://mainnet-rpc.thundercore.io"

Reply:
```suggestion
      "https://mainnet-rpc.thundercore.com/archived/SNAPSHOTEuR82a75fLYA"
      "https://mainnet-rpc.thundercore.com"
      "https://mainnet-rpc.thundertoken.net"
      "https://mainnet-rpc.thundercore.io"
```

Reply:
Archive node is attached in the array of rpc